### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by smk4664**
This PR adds in the new Doc Standards for Nautobot Apps.

### CODE REVIEW

File: docs/admin/release_notes/version_1.7.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# v1.7 Release Notes
+
+## [v1.7.0](https://github.com/nautobot/nautobot-plugin-chatops/releases/tag/v1.7.0)
+
+### Additions
+
+[#128](https://github.com/nautobot/nautobot-plugin-chatops/pull/128)
```
Comment (itdependsnetworks) [inline docs/admin/release_notes/version_1.7.md:7]: Can you add a description?

**Comment by itdependsnetworks** [comment]
How does one run this? I do not see docker-compose.dev.yml being updated to run the docs container.

File: pyproject.toml [hunk 2]
```diff
@@ -36,6 +39,16 @@ pydocstyle = "*"
 prybar = "*"
 invoke = "*"
 flake8 = "*"
+# Allow Markdown files to include other files
```
Comment (itdependsnetworks) [inline pyproject.toml:42]: Would follow pattern done here: https://github.com/nautobot/nautobot-plugin-golden-config/pull/325/files#diff-7db2fc97b6c21ab7963634d35cf5dc9b91e804f1870046d0f9db5b32148d86ae

**Comment by smk4664** [comment]
> How does one run this? I do not see docker-compose.dev.yml being updated to run the docs container.

Fixed, added the docs container.

**Comment by itdependsnetworks** [comment]
see: https://github.com/nautobot/nautobot/pull/2365

**Comment by whitej6** [comment]
Other than resolving failing poetry issue on tests LGTM

**Comment by smk4664** [comment]
Thanks! That seems to be the hard part though.

**Comment by whitej6** [comment]
I wonder if dropping 1.1 support of Nautobot would help

**Review by whitej6 — APPROVED** [review]

**Pull request merged**